workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -177,5 +166,18 @@ public void CreateActivationManagerAndReaderWriterStore(Entity entity)
             var manager = new MonoBehaviourActivationManager(gameObject, injector, store, logger);
             entityToActivationManager.Add(entity, manager);
         }
+
+        internal void RemoveActivationManagerAndReaderWriterStore(Entity entity)
+        {
+            if (!entityToActivationManager.ContainsKey(entity))
```

Comment:
Use `TryGet` to avoid a double lookup here and below.

Reply:
Fixed. :)